pulp_rpm/app/viewsets.py

```diff
@@ -245,6 +495,72 @@ class RpmPublicationViewSet(PublicationViewSet):
     endpoint_name = "rpm"
     queryset = RpmPublication.objects.exclude(complete=False)
     serializer_class = RpmPublicationSerializer
+    queryset_filtering_required_permission = "rpm.view_rpmpublication"
+
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["list"],
+                "principal": ["authenticated"],
+                "effect": "allow",
+                "condition": "has_model_perms:rpm.view_rpmpublication",
+            },
+            {
+                "action": ["retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_model_or_obj_perms:rpm.view_rpmpublication",
+            },
+            {
+                "action": ["create"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_model_perms:rpm.add_rpmpublication",
```

Comment:
you should also consider permissions to view the repo which is used to create the publication